cartographer/mapping/internal/3d/scan_matching/rotational_scan_matcher_test.cc

```diff
@@ -28,7 +28,8 @@ namespace {
 TEST(RotationalScanMatcher3DTest, OnlySameHistogramIsScoreOne) {
   Eigen::VectorXf histogram(7);
   histogram << 1.f, 43.f, 0.5f, 0.3123f, 23.f, 42.f, 0.f;
-  RotationalScanMatcher matcher({{histogram, 0.f}});
+  RotationalScanMatcher matcher(
```

Comment:
Why this change?

Reply:
oh, ic, that's because you'll eventually remove this constructor. I'd move this change to the other PR then, where you actually remove the constructor.

Reply:
Okay, I reverted the change. Now it has the same semantics as before, but I still have to update the tests, otherwise the compiler complains about ambiguous constructors for `RotationalScanMatcher`.